Property 'database' does not exist on type 'FirebaseNamespace'

<!-- Step 1 [READ THIS] -->
<!--
Are you in the right place?
  * For issues or feature requests related to __the code in this repository__
    file a Github issue.
    * If this is a __feature request__ make sure the issue title starts with "FR:".
  * For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/)
    with the firebase tag.
  * For general Firebase discussion, use the [firebase-talk](https://groups.google.com/forum/#!forum/firebase-talk)
    google group.
  * For help troubleshooting your application that does not fall under one
    of the above categories, reach out to the personalized
    [Firebase support channel](https://firebase.google.com/support/).
-->

<!-- Step 2 -->

### [REQUIRED] Describe your environment

  * Operating System version:
    * Distributor ID:	Ubuntu
    * Description:	Ubuntu 18.04 LTS
    * Release:	18.04
    * Codename:	bionic
  * Firebase SDK version: 5.0.1
  * Firebase Product: database

<!-- Step 3 -->

### [REQUIRED] Describe the problem

#### Steps to reproduce:
<!--
  What happened? How can we make the problem occur?
  This could be a description, log/console output, etc.
-->
```
import firebase from 'firebase/app';
import 'firebase/database';

...

in code
firebase.database().ref(userRef).once('
...
```
causes following error.

error TS2339: Property 'database' does not exist on type 'FirebaseNamespace'.

#### Workaround
When I changed my code like following solves issue.

```
firebase['database']().ref(userRef).once('
```

#### Relevant Code:

<!-- 
  Reproduce the issue on StackBlitz and provide your forked URL
  or give us some sample code below
-->


```javascript
// TODO(you): code here to reproduce the problem
```

--- a/packages/firebase/.gitignore
+++ b/packages/firebase/.gitignore
@@ -2,3 +2,4 @@
 /firebase*.map
 /firebase*.gz
 /firebase*.tgz
+*/*.d.ts

--- a/packages/firebase/tsconfig.json
+++ b/packages/firebase/tsconfig.json
@@ -2,7 +2,6 @@
   "extends": "../../config/tsconfig.base.json",
   "compilerOptions": {
     "outDir": "dist",
-    "declaration": false
   },
   "exclude": [
     "dist/**/*"

